### DISCUSSION TITLE
Ossec Apparmor Denied Logspam

### DISCUSSION THREAD

**Discussion by Jus-R** [Security-Onion-Solutions/securityonion#2543] [Unsupported Versions] [answered]
Ossec creates ~5700 Daily Alerts for Apparmor Denied.

The Message looks like this:
{"timestamp":"2021-01-05T20:26:30.803+0000","rule":{"level":3,"description":"Apparmor DENIED","id":"52002","firedtimes":105,"mail":false,"groups":["local","syslog","apparmor"]},"agent":{"id":"001","name":"xxx","ip":"xxx"},"manager":{"name":"xxx-wazuh-manager"},"id":"1609878390.2230765","full_log":"Jan  5 20:26:30 xxx kernel: [2955743.701286] audit: type=1400 audit(1609878390.062:96652): apparmor=\"DENIED\" operation=\"ptrace\" profile=\"docker-default\" pid=26439 comm=\"telegraf\" requested_mask=\"trace\" denied_mask=\"trace\" peer=\"unconfined\"","predecoder":{"program_name":"kernel","timestamp":"Jan  5 20:26:30","hostname":"xxx"},"decoder":{"parent":"kernel","name":"kernel"},"data":{"status":"DENIED","extra_data":"ptrace"},"location":"/var/log/syslog"}

Which also creates an entry in /var/log/kern.log and syslog looking like this:

audit: type=1400 audit(1608077130.299:36612): apparmor="DENIED" operation="ptrace" profile="docker-default" pid=14562 comm="telegraf" requested_mask="trace" denied_mask="trace" peer="unconfined"

Now this creates a load of entries in those logs and unecessary spam in the alert view.
It seems similar to an older docker issue referenced here : https://github.com/moby/moby/issues/7276

Is this a known issue or something specific to my installation as i haven't seen any similar post so far?

**Comment by TOoSmOotH** [comment, marked as answer]
Created #2560 to address this.
